internal/exec/resolvable/packer.go

```diff
@@ -130,8 +130,11 @@ func (b *execBuilder) makeStructPacker(values common.InputValueList, typ reflect
 	var fields []*structPackerField
 	for _, v := range values {
 		fe := &structPackerField{field: v}
+		fx := func(n string) bool {
+			return strings.EqualFold(n, v.Name.Name) || strings.EqualFold(camelize(n), camelize(v.Name.Name))
```

Comment:
@neelance Yeah, you're right :). Updated in https://github.com/neelance/graphql-go/pull/77/commits/6e8c113057e01a51ddc13f17702b4d3506c195e7.